reinforcement_learning/bandits_recsys_movielens_testbed/bandits_movielens_testbed.ipynb

```diff
@@ -0,0 +1,748 @@
+{
```

Comment:
<p>For the added three features, I understand 2 and 3 well. For the bullet point 1,  </p><blockquote>The number of actions available to the agent can change over time.&nbsp;</blockquote><p>Does the notebook showcase this feature as well? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/949/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>